src/libs/Navigation/AppNavigator/Navigators/FullScreenNavigator.tsx

```diff
@@ -20,17 +39,19 @@ function FullScreenNavigator() {
 
     return (
         <View style={styles.rootNavigatorContainerStyles(isSmallScreenWidth)}>
-            <RootStack.Navigator>
+            <RootStack.Navigator screenOptions={screenOptions.fullScreenInner}>
```

Comment:
I can't understand why we create `screenOptions.fullScreenInner` instead of using `screenOptions.centralPaneNavigator`